src/assets/scripts/client/App.js

```diff
@@ -50,8 +51,55 @@ export default class App {
         this.prop.log = LOG.DEBUG;
         this.prop.loaded = false;
 
-        return this.setupHandlers()
-            .loadInitialAirport(airportLoadList, initialAirportToLoad);
+        $.getJSON('assets/airports/airportLoadList.json')
+            .done((data) => {
+                // List of airports to load
```

Comment:
I love comments (as I'm sure you've seen already).  However, I question the value of this comment.  Does this tell us anything the (well named) variable does not?

Reply:
This is what I get for blindly shunting the JSDoc comment into the code without using my head (this was originally a param passed to the constructor)